sdk/go/pulumi/mocks.go

```diff
@@ -33,6 +33,7 @@ import (
 )
 
 type MockResourceMonitor interface {
+	Invoke(args MockCallArgs) (resource.PropertyMap, error)
 	Call(args MockCallArgs) (resource.PropertyMap, error)
```

Comment:
Oh this is unfortunate... this is public API that users could be writing tests against so I don't think we can change the meaning of Call. If it was being used for invokes it needs to keep being used for invokes.